packages/web-components/fast-element/src/attributes.ts

```diff
@@ -56,19 +103,65 @@ export const nullableNumberConverter: ValueConverter = {
     },
 };
 
+/**
+ * An implementation of {@link Accessor} that supports reactivity,
+ * change callbacks, attribute reflection, and type conversion for
+ * custom elements.
+ * @public
+ */
 export class AttributeDefinition implements Accessor {
     private readonly fieldName: string;
     private readonly callbackName: string;
     private readonly hasCallback: boolean;
     private readonly guards: Set<unknown> = new Set();
 
+    /**
+     * The class constructor that owns this attribute.
+     */
+    public readonly Owner: Function;
+
+    /**
+     * The name of the property associated with the attribute.
+     */
+    public readonly name: string;
+
+    /**
+     * The name of the attribute in HTML.
+     */
+    public readonly attribute: string;
+
+    /**
+     * The {@link AttributeMode} that describes the behavior of this attribute.
+     */
+    public readonly mode: AttributeMode;
+
+    /**
+     * A {@link ValueConverter} that integrates with the property getter/setter
+     * to convert values to and from a DOM string.
+     */
+    public readonly converter?: ValueConverter;
+
+    /**
+     *
```

Comment:
Should we put something like "Creates an instance of AttributeDefinition"?

Reply:
Oops ;) On it.